tests/unit/engine/providers/data/APIProxy_data.py

```diff
@@ -0,0 +1,193 @@
+            },
+            'version': '1.3.0dev'
+        },
+        {
+            'kind': 'infrastructure/machine',
+            'title': 'Virtual Machine Infra',
+            'provider': 'any',
+            'name': 'service-1',
+            'specification': {
+                'ip': '20.73.105.54',
+                'hostname': 'service-vm-4'
+            },
+            'version': '1.3.0dev'
+        },
+        {
+            'kind': 'infrastructure/machine',
+            'title': 'Virtual Machine Infra',
+            'provider': 'any',
+            'name': 'service-2',
+            'specification': {
+                'ip': '20.73.105.188',
+                'hostname': 'service-vm-1'
+            },
+            'version': '1.3.0dev'
+        },
+        {
+            'kind': 'infrastructure/machine',
+            'title': 'Virtual Machine Infra',
+            'provider': 'any',
+            'name': 'service-3',
+            'specification': {
+                'ip': '20.73.105.240',
+                'hostname': 'service-vm-0'
+            },
+            'version': '1.3.0dev'
+        },
+        {
+            'kind': 'infrastructure/machine',
+            'title': 'Virtual Machine Infra',
+            'provider': 'any',
+            'name': 'service-4',
+            'specification': {
+                'ip': '20.73.105.33',
+                'hostname': 'service-vm-3'
+            },
+            'version': '1.3.0dev'
+        },
+    ])
+
+
+def CLUSTER_MODEL(provider: str) -> ObjDict:
+    return dict_to_objdict({
+        'kind': 'epiphany-cluster',
+        'title': 'Epiphany cluster Config',
+        'provider': f'{provider}',
+        'name': 'default',
+        'specification': {
+            'prefix': 'prefix',
+            'name': 'cluster',
+            'admin_user': {
+                'name': 'username',
+                'key_path': '/path/to/key'
+            },
+            'cloud': {
+                'k8s_as_cloud_service': False,
+                'subscription_name': 'Subscription Name',
+                'vnet_address_pool': '10.1.0.0/20',
+                'use_public_ips': True,
+                'use_service_principal': False,
+                'region': 'West Europe',
+                'network': {'use_network_security_groups': True},
+                'default_os_image': 'default'
+            },
+            'components': {
+                'service': {
+                    'count': 5,
+                    'machine': 'service-machine',
+                    'configuration': 'default',
+                    'subnets': [{'address_pool': '10.1.8.0/24'}],
+                    'machines': ['service-0',
+                                 'service-1',
+                                 'service-2',
+                                 'service-3',
+                                 'service-4']
+                }
+            }
+        },
+        'version': '1.3.0dev'
+    })
+
+RUNNING_INSTANCES: List[List[Dict]] = [
```

Comment:
This would be specific to Azure, maybe name it something like RUNNING_INSTANCES_AZURE?